### ISSUE TITLE
Design a "subscribe container" for the homepage and contact page so that users can choose to be notified whenever a new piece is posted on the site 

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/mep-django#454]
1. Design the placement of the feature on the homepage and the contact pages.
2. Create a design for the feature so that it will match the current visual identity of the project 
3. 1 and 2 will be applied to desktop, tablet, and mobile.

**Comment by gissoo** [comment]
Designed a total of 6 screens: 
1. Homepage 
a. Desktop
b. Tablet 
c. Mobile

2. Contact us
a. Desktop
b. Tablet 
c. Mobile

@jkotin Please review the designs here: "https://app.zeplin.io/project/5c48e7313d2b13bf7fc9fe14/screen/5de5723a1d3db37beaa8a66e"

**Comment by gissoo** [comment]
@jkotin Please also review this link: "https://app.zeplin.io/project/5c48e7313d2b13bf7fc9fe14/screen/5de6871d61773d4859fc7b40" 

This is the behavior of the text box and the button when the user interacts with it. It is designed to be subtle while being responsive to the users' actions for improved usability.

**Comment by jkotin** [comment]
Did we defer this issue to a later release?  I think the designs look good.  I have some ideas about language.  E.g. we might want "join" instead of "subscribe." Should I respond fully to the issue now or wait until the next phase of testing? @gissoo

**Comment by gissoo** [comment]
@jkotin yes we did! I would suggest let's wait on this, still need to figure out where it's currently best to record language needed from you.

**Comment by rlskoeser** [comment]
From my perspective: we deferred the implementation, but you could sign off on the design if it's ready and you're happy with it. I'm not sure language changes are sufficient for the design not to be approved, what do you think @gissoo ? 

We do continue to have the problem with managing this "microcopy" content...

**Comment by thatbudakguy** [comment]
chiming to say that it would awesome if we could, more broadly, handle the problem of "microcopy" natively in wagtail (perhaps using snippets).

**Comment by gissoo** [comment]
@rlskoeser I have a feeling some things including the language are not final yet and may change about this since we didn't fully spend time talking about this yesterday and decided to defer it to a later release. I want to make sure the designs stay up-to-date, my instinct tells me to wait on this design issue as well till it's final.

**Comment by rlskoeser** [comment]
@gissoo ok, that's fine with me - defer to you on this. Would you put it back in the appropriate status (not sure what makes the most sense - icebox, maybe, since we're deferring it for now?) and remove the awaiting review label?
